app/nexus/internal/route/health/status.go

```diff
@@ -0,0 +1,432 @@
+package health
+
+import (
+	"context"
+	"crypto/fips140"
+	"crypto/tls"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/spiffe/go-spiffe/v2/workloadapi"
+	"github.com/spiffe/spike-sdk-go/api/entity/data"
+	apiErr "github.com/spiffe/spike-sdk-go/api/errors"
+
+	"github.com/spiffe/spike-sdk-go/api/entity/v1/reqres"
+	"github.com/spiffe/spike-sdk-go/log"
+	env "github.com/spiffe/spike/app/nexus/internal/env"
+	state "github.com/spiffe/spike/app/nexus/internal/state/base"
+	"github.com/spiffe/spike/internal/journal"
+	"github.com/spiffe/spike/internal/net"
+)
+
+// startTime records when the server was started, used for uptime calculation
+var startTime = time.Now()
+
+// RouteGetStatus handles GET requests to the /v1/operator/status endpoint.
+// It performs the following steps:
+// Audits the incoming request for monitoring and compliance purposes.
+// Guards the request by validating the SPIFFE ID and checking ACL permissions.
+// Fetches the current system status, including keeper status, root key status, backing store health, FIPS mode, secrets count, and overall health.
+// Marshals the aggregated system status into JSON and responds to the client.
+// Handles errors gracefully, returning appropriate HTTP status codes and messages.
+func RouteGetStatus(
+	w http.ResponseWriter, r *http.Request, audit *journal.AuditEntry,
+) error {
+	const fName = "routeGetStatus"
+	// Audit
+	journal.AuditRequest(fName, r, audit, journal.AuditRead)
+
+	requestBody := net.ReadRequestBody(w, r)
+
+	// HandleRequest pattern
+	request := net.HandleRequest[
+		reqres.HealthReadRequest, reqres.HealthReadResponse](
```

Comment:
@doguhanniltextra can you add these to SPIKE Go SDK.

It's better to cut a release there, refer them here and then continue review, I think.

Thanks a lot 🙏 .

Please let me know when you create the SDK PR, so I can cut a minor release.